template-processors/base/bin/processParameters.sh

```diff
@@ -21,45 +21,48 @@ echo "Processing Parameters"
 
 FOLDERS=""
 
-if [ -e ${CLONED_PARAMETER_GIT_DIR}/hiera.lst ] ; then
+if [ -e "${CLONED_PARAMETER_GIT_DIR}/hierarchy.lst" ] ; then
   echo "Generating hierarchy"
   # a hierarchy list was provided, so lets process it
-  while read -r DIR
-    do
+  while read -r DIR ; do
```

Comment:
Please add `IFS=` before `read` — it adds extra protection against squashing of whitespaces in the dir names when doing `read -r` — i.e.:

    while IFS= read -r DIR; do